en/flight_controller/beaglebone_blue.md

```diff
@@ -17,118 +17,114 @@ Information about flashing OS images can be found on [this page](https://github.
-1. Install pre-built native (ARM) version of librobotcontrol.\* into `$LIBROBOTCONTROL_INSTALL_DIR/lib`
-
-At this point the BeagleBone Blue target can be built on both cross compile host system and native build system, i.e., 
-```sh
-make posix_bbblue_cross [upload]
-```
-
-## Cross Compiler Build
+## Cross Compiler Build (Recommend)
 
 The recommended way to build PX4 for *BeagleBone Blue* is to compile on a development computer and upload the PX4 executable binary directly to the BeagleBone Blue. 
 
 > **Tip** This approach is recommended over [native build](#native_builds) due to speed of deployment and ease of use.
 
-### Cross Compiler Setup
+### Beaglebone Blue WIFI Setup
 
-ARM Cross Compiler for *BeagleBone Blue* can be found at [Linaro Toolchain Binaries site](http://www.linaro.org/downloads/). 
-The following is an example setup on development host. 
+For easy access to your board, you can connect it to your home network via wifi.
 
-1. First install the toolchain into */opt/bbblue_toolchain/gcc-arm-linux-gnueabihf*. Here is an example of using soft link to select which version of the toolchain you want to use:
-   ```sh
-    ...@ubuntu:/opt/bbblue_toolchain$ ls -l
-        lrwxrwxrwx 1 root root   51 Mar 22 16:10 gcc-arm-linux-gnueabihf -> gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf
-        drwxr-xr-x 8 root root 4096 May 17  2017 gcc-linaro-6.3.1-2017.05-x86_64_arm-linux-gnueabihf
-        drwxr-xr-x 8 root root 4096 Nov 19 03:51 gcc-linaro-6.4.1-2017.11-x86_64_arm-linux-gnueabihf
-   ```
-    
-1. Add it to the PATH in ~/.profile as shown below
-   ```sh
-   export PATH=$PATH:/opt/bbblue_toolchain/gcc-arm-linux-gnueabihf/bin
-   export CrossCompiler=/opt/bbblue_toolchain/gcc-arm-linux-gnueabihf/bin/arm-linux-gnueabihf-
-   ```
+The steps are:
+```
+sudo -su
+connmanctl
+connmanctl>scan wifi
+connmanctl>services(at this point you should see your network appear along with other stuff, in my case it was "AR Crystal wifi_f45eab2f1ee1_6372797774616c_managed_psk")
+connmanctl>agent on
+connmanctl>connect wifi_f45eab2f1ee1_6372797774616c_managed_psk
+connmanctl>quit
+```
+### SSH root Login on Beaglebone
+With USB Serial connectet 
+```
+echo "PermitRootLogin yes" >>  /etc/ssh/sshd_config && systemctl restart sshd
+```
 
-> **Tip** GCC in the toolchain should be compatible with kernel in *BeagleBone Blue*. General rule of thumb is to choose a toolchain 
-where version of GCC is not higher than version of GCC which comes with the OS image on *BeagleBone Blue*. 
+### Cross Compiler Setup
 
+1. First set up *rsync* (this is is used to transfer files from the development computer to the target board over a network - WiFi or Ethernet). 
+   For *rsync* over SSH with key authentication, follow steps here:
+   1. sudo su
+   1. ssh-keygen -t rsa -C root@beaglebone
+   1. ENTER //no passphrase
+   1. ENTER
+   1. ENTER
+   1. ssh-copy-id root@beaglebone
```

Comment:
Where is this executed?

Reply:
Cross Compiler Machine

Reply:
@bkueng It looks like all of this is run on the development machine - inferred from the heading? Or did you mean something else?

Reply:
This is not correct then.
- Why would you do this as root?
- `ssh-keygen -t rsa -C root@beaglebone` might overwrite an existing key

Reply:
Hi @Superbear88

Did you see this response from @bkueng ?